sklearn/ensemble/tests/test_base.py

```diff
@@ -54,8 +54,7 @@ def test_base_zero_n_estimators():
     # a ValueError.
     ensemble = BaggingClassifier(base_estimator=Perceptron(), n_estimators=0)
     iris = load_iris()
-    err_msg = "n_estimators must be greater than zero, got 0."
-    with pytest.raises(ValueError, match=err_msg):
+    with pytest.raises(ValueError):
```

Comment:
We could remove these tests but i'm ok to keep them for now while there's still estimator validation in BaseEnsemble.

Reply:
Ok great